index.html

```diff
@@ -97,18 +107,21 @@ <h2 class="h2-message">Leave a Message</h2>
     </form>
     </div>
     </section>
-<!--Messages submitted-->
+
+    <!--Messages submitted-->
     <section id="messages">
        <div>
        <h2 class="h2-message">Messages</h2>
            <ul></ul>
        </div>
     </section>    
- <section>
+
+    <!--CONNECT SECTION-->
+    <section id = "connect">
  <h1 class="h1-section">Connect</hi>
-  <h3> Email: gtturumella@gmail.com &nbsp; GitHub
-  <a href="https://github.com/Tugeeta"><img src="github.jpg" width="25"> &nbsp; &nbsp; Linkedin</a>
-  <a href="https://www.linkedin.com/in/geeta-devi-404679238/"><img src="linkedin.jpg" width="25"></a>
+  <h3> <a href ="mailto:gtturumella@gmail.com"> Email &nbsp; </a>
+  <a href="https://github.com/Tugeeta">GitHub<img src="github.jpg" width="25"> &nbsp; &nbsp;</a>
+  <a href="https://www.linkedin.com/in/geeta-devi-404679238/">Linkedin<img src="linkedin.jpg" width="25"></a>
```

Comment:
You probably don't want trailing spaces inside your links: It looks looks weird when you hover.